packages/apps/storybook/src/examples/ProjectAwareComponent.tsx

```diff
@@ -1,13 +1,13 @@
 /*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
 
 import React from "react";
 
 export interface ProjectAwareComponentProps {
-  /** Project to know about */
-  projectId: string | undefined;
+  /** iTwin to know about */
```

Comment:
Might want to rename the interface and component.

Reply:
Done